test/parallel_api/algorithm/alg.sorting/sort.pass.cpp

```diff
@@ -1,5 +1,5 @@
 // -*- C++ -*-
-//===-- sort.pass.cpp -----------------------------------------------------===//
```

Comment:
Why was this name removed?

Reply:
Removed by review comment from @MikeDvorskiy